### DISCUSSION TITLE
Is there a way to add a mount to an existing podman machine on macOS?

### DISCUSSION THREAD

**Discussion by kgibm** [containers/podman#14774] [Q&A] [answered]
Hi, I know that I can create a mount when initializing a podman machine, e.g.:

```
podman machine init --cpus 4 --memory 10240 --disk-size 100 -v $HOME:/mnt/host
```

Is there a way to add a mount to an existing podman machine? I would rather not delete and re-create my existing podman machine as it has a lot of custom-built images inside.

**Comment by baude** [comment, marked as answer]
we are working on dynamic mounts, so there is nothing quite yet.  I do *think* that if you edit the json file manually, you should be able to add an additional mount.  this is technically not supported but should work.  you will want to back up the file and make sure the machine is shutdown.  to find the file and path, do a `podman machine rm` and then select `No` to the are you sure question.  the file you want is .json.

**Reply by kgibm** [reply]
That works, thanks! For me, it was in `~/.config/containers/podman/machine/qemu/podman-machine-default.json`

**Reply by ssbarnea** [reply]
The current default location seems to be ` ~/.config/containers/podman/machine/applehv/podman-machine-default.json` with podman v5.

**Reply by baude** [reply]
my answer does not apply to podman 5 as macos now uses virtiofs ... and there needs to be some changes on the podman machine side when  a mount is done fwiw
